damus/Views/CreateAccountView.swift

```diff
@@ -167,7 +167,7 @@ func FormLabel(_ title: String, optional: Bool = false) -> some View {
         Text(title)
                 .bold()
         if optional {
-            Text("- optional", comment: "Label indicating that a form input is optional.")
```

Comment:
The `-` makes it a bit harder for translators to translate to other languages, and it isn't really necessary in the UI.